.github/workflows/contracts.yml

```
@@ -196,7 +196,9 @@ jobs:
         env:
           CHAIN_API_URL: ${{ secrets.SEPOLIA_ETH_HOSTNAME_HTTP }}
           ACCOUNTS_PRIVATE_KEYS: ${{ secrets.TESTNET_ETH_CONTRACT_OWNER_PRIVATE_KEY }}
-        run: yarn deploy --network ${{ github.event.inputs.environment }}
+        env:
```

actionlint (1.7.12)

[error] key "env" is duplicated in element of "steps" section. previously defined at line:196,col:9

(syntax-check)